data-es7.js

```diff
@@ -738,7 +763,10 @@ exports.tests = [
   category: 'strawman',
   link : 'https://github.com/DavidBruant/Map-Set.prototype.toJSON',
   exec: function(){/*
-    return JSON.stringify(new Map([['a', 'b'], ['c', 'd']])) === '[["a","b"],["c","d"]]';
+    var map = new Map();
+    map.set('a', 'b');
+    map.set('c', 'd');
+    return JSON.stringify(map) === '[["a","b"],["c","d"]]';
```

Comment:
It's not about that, since the compatibility table is testing polyfills and shims too. Since `Map`/`Set` *has* been shipped such that it does not support initializations from iterables, the test should not break in those browsers.

As for adding `es6-shim` as a dependency for the `es7-shim` for the compatibility table, that's not a bad approach, but I'm not sure that's warranted. There's zero reason to keep these tests compact.